### PR TITLE
CLDR-16534 Downgrade paths for islamic calendar display names (renaming to use Hijri)

### PR DESCRIPTION
**Pull request by pedberg-icu**
CLDR-16534

- [ ] This PR completes the ticket.

One part of changing the islamic calendar display names to use Hijri: Downgrade the relevant paths.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

File: tools/cldr-code/src/main/resources/org/unicode/cldr/util/downgrade.txt [hunk 1]
```diff
@@ -12,6 +12,9 @@
 
 # CLDR-16228 Add the paths where spaces changed in cjk+ person names
 # CLDR-16568 Require checking of questionable PNF data
+# CLDR-16534 Downgrade paths for renaming Islamic Calendar to Hijri Calendar
```
Comment (macchiati) [inline tools/cldr-code/src/main/resources/org/unicode/cldr/util/downgrade.txt:15]: I think we want to move this up, so the comment about PNF data is just before the PNF paths

**Review by macchiati — CHANGES_REQUESTED** [review]

File: tools/cldr-code/src/main/resources/org/unicode/cldr/util/downgrade.txt (hunk 1 above)
Comment (btangmu) [reply tools/cldr-code/src/main/resources/org/unicode/cldr/util/downgrade.txt:15]: I've just merged the commit that moves this file from util to util/data

Comment (pedberg-icu) [reply tools/cldr-code/src/main/resources/org/unicode/cldr/util/downgrade.txt:13]: > I think we want to move this up, so the comment about PNF data is just before the PNF paths

@macchiati I rearranged the lines as requested (I think)

Comment (btangmu) [reply tools/cldr-code/src/main/resources/org/unicode/cldr/util/downgrade.txt:13]: yes, different topic, however it's the same file so potential merge conflict

Comment (pedberg-icu) [reply tools/cldr-code/src/main/resources/org/unicode/cldr/util/downgrade.txt]: @macchiati OK I have resolved conflicts, please approve, thanks

**Pull request merged**